integration-tests/jest-puppeteer.config.js

```diff
@@ -1,49 +1,11 @@
-/* eslint-disable prefer-template */
-const printWarning = (message) => {
-  if (process.env.warningPrinted) {
-    return;
-  }
-  console.warn('\n\x1b[1m' /* bold */ + '╔'.padEnd(80, '═') + '╗');
-  message.forEach((line) => {
-    console.warn(`║ ${line}`.padEnd(80) + '║');
-  });
-  console.warn('╚'.padEnd(80, '═') + '╝');
-  console.warn('\x1b[0m' /* not-bold */);
-  process.env.warningPrinted = true;
-};
-
 module.exports = {
   browserContext: 'incognito',
   launch: {
     devtools: process.env.DEVTOOLS === 'true',
     headless: process.env.HEADLESS !== 'false',
     slowMo: process.env.SLOW === 'true' ? 250 : undefined,
     defaultViewport: { width: 1200, height: 800 },
-    // Workaround for issue when accessing cross domain iframes in puppeteer while not headless:
-    // https://github.com/GoogleChrome/puppeteer/issues/4960
-    args: (({ headfull, devtools }) => {
-      const flag = '--disable-features=site-per-process';
```

Comment:
None of this is needed anymore-- I ran the IA analysis test locally (which interacts with an iframe) with both HEADFUL and DEVTOOLS, and it passed.